### PR TITLE
Add a new check which allows checking for extra config

### PR DESCRIPTION
**Pull request by jibsheet**
@edx/devops Please have a look

### CODE REVIEW

File: playbooks/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 ---
+- name: Check Configuration Sources
+  fail:
+    msg: "Configuration Sources Checking is enabled, you must defined {{ item }}"
```
Comment (maxrothman) [inline playbooks/roles/common/tasks/main.yml]: typo: "defined"

Comment (maxrothman) [inline playbooks/roles/common/tasks/main.yml]: This error message might appear cryptic to someone who didn't know much about the config sources checking system. I'd say the comment in defaults should be duplicated here, or you should have a pointer to it, or something.

**Review by maxrothman — COMMENTED** [review]
I'll give this a I-can't-believe-this-works-but-I-guess-we'll-ship-it-anyways thumb on general functionality 😞 👍

**Comment by jibsheet** [comment]
@maxrothman Pushed a documentation update, let me know if it makes sense and I'll squash so I can merge and move forward with updating all the private repos.

**Comment by maxrothman** [comment]
👍

**Pull request merged**